src/Operator/Base.jl

```diff
@@ -371,12 +373,17 @@ function getelementmatrix(operator::Operator)
             Btcurrent = []
             for mode in output.evaluationmodes
                 if mode == EvaluationMode.interpolation
-                    numberfields += output.basis.numbercomponents
+                    numberfields += 1
+                    numbernodeoutputs += output.basis.numbercomponents
+                    numberquadratureoutputs += output.basis.numbercomponents
                     Btcurrent =
                         Btcurrent == [] ? output.basis.interpolation :
                         [Btcurrent; output.basis.intepolation]
                 elseif mode == EvaluationMode.gradient
-                    numberfields += output.basis.dimension * output.basis.numbercomponents
+                    numberfields += output.basis.dimension
```

Comment:
It seems my `numberfields` change here was wrong, but I don't understand why it has a different meaning here than for inputs (line 324, 331).

Reply:
I pushed a fixup (8a9f245b6979b4b642f2cfaf048cfad3705821ef) and tests pass locally now, but @jeremylt can you compare with the above lines and sign off on this bit, or maybe the input handling needs an update?

Reply:
I think this is good